acceptance/analyzer_test.go

```diff
@@ -1054,6 +1068,17 @@ func assertAnalyzedMetadata(t *testing.T, path string) {
 	h.AssertNil(t, err)
 }
 
+func getAnalyzedMetadataImageRef(t *testing.T, path string) *platform.AnalyzedMetadata {
```

Comment:
```suggestion
func getAnalyzedMetadata(t *testing.T, path string) *platform.AnalyzedMetadata {
```

Reply:
(since it returns the whole metadata)

But maybe we could modify `assertAnalyzedMetadata` (which does essentially the same thing) to return the metadata struct.

Reply:
Fixed in e1f3c71